### DISCUSSION TITLE
Option to open PDF native in Browser

### DISCUSSION THREAD

**Discussion by gruselglatz** [gotson/komga#1631] [Ideas]
Would it be possible to add the option to open PDFs native in Browser, or some addition in the browser reader to render pdfs as what they are?
This would allow for searching, selecting and opening Links which are embedded.

We would like to use Komga mainly for reading Magazines.

Thanks a lot :)

**Comment by gotson** [comment]
click the download link?

**Reply by gruselglatz** [reply]
This downloads the PDF, yes this is a workaround.

Its not the same as a reader which stores my progress and more.

**Reply by gotson** [reply]
if it was using the browser pdf reader it would not be able to sync progress, so it's basically the same thing as downloading and using the desktop pdf viewer
